src/screenshots/index.js

```diff
@@ -69,4 +70,10 @@ export default class Screenshots {
 
         return new Capturer(this.screenshotsPath, testEntry, connection, pathPattern, this.fullPage, warningLog);
     }
+
+    watchForTestRun (test, testRun, connection) {
+        const testEntry = this._ensureTestEntry(test);
```

Comment:
I guess it can be simplified as follows:
`const testEntry = this._getTestEntry(test);` 
Because `this._ensureTestEnty` method was called into `this.screenshots.createCapturerFor` method.